src/neoxp/Extensions/Extensions.cs

```diff
@@ -57,6 +58,28 @@ public static void WriteWarning(string value)
             Console.WriteLine($"\x1b[33mWarning: {value}\x1b[0m");
         }
 
+        public static bool TryGetUtf8String(this ReadOnlySpan<byte> data, out string text)
+        {
+            try
+            {
+                text = Encoding.UTF8.GetString(data);
```

Comment:
Wouldn't be easier to use Strict UTF-8?

Reply:
It will get very low perfomance to catch the exception when using random `data` with Strict UTF-8. `try-catch` here is use for robust, and I don't expect it runs into `catch` case.